discovery-provider/plugins/network-monitoring/src/metrics/queries.ts

```diff
@@ -206,3 +206,21 @@ export const getUnsyncedUsersCount = async (run_id: number): Promise<number> =>
     return usersCount
 }
 
+// The number of users whose primary content node is null
+export const getGhostUsersCount = async (run_id: number): Promise<number> => {
```

Comment:
can we give this a more descriptive name throughout - just `usersWithNoPrimary` or smth